Regression/WarpX-tests.ini

```
@@ -2103,6 +2103,21 @@ compileTest = 0
 doVis = 0
 compareParticles = 0
 
+[embedded_boundary_rotated_cube]
+buildDir = .
+inputFile = Examples/Modules/embedded_boundary_rotated_cube/inputs_3d
+runtime_params =
+dim = 3
+addToCompileString = USE_EB=TRUE
+restartTest = 0
+useMPI = 1
+numprocs = 1
+useOMP = 1
+numthreads = 1
+compileTest = 0
+doVis = 0
+compareParticles = 0
```

I think if you don't add this, no Python analysis (not even the default regression analysis) is performed:
```suggestion
compareParticles = 0
analysisRoutine = Examples/Modules/embedded_boundary_rotated_cube/analysis_fields.py
```

Actually I think a similar line is missing for the previous embedded boundary test `embedded_boundary_cube` as well, where I think you would need to add
```
analysisRoutine = Examples/Modules/embedded_boundary_cube/analysis_fields.py
```